.github/workflows/release.yml

```diff
@@ -383,18 +383,17 @@ jobs:
       run: |
         tar -xf source.tar.gz --strip-components=1
 
-    # The following step can be removed and the default vcpkg installation
-    # (/usr/local/share/vcpkg) restored when the build VM is updated with a revision of
+    # The following step can be removed when the build VM is updated with a revision of
     # vcpkg dating from roughly 01/01/2021 or later.
     - name: Update vcpkg
       run: |
-        cd /tmp
-        git clone https://github.com/Microsoft/vcpkg
+        cd /usr/local/share/vcpkg
+        git fetch --unshallow
 
     - name: Prepare vcpkg (with cache)
       uses: lukka/run-vcpkg@v6
       with:
-        vcpkgDirectory: '/tmp/vcpkg'
+        vcpkgDirectory: '/usr/local/share/vcpkg'
         doNotUpdateVcpkg: false
```

Comment:
Can this also be put back on `true` in this case?

Reply:
We do need it to update to the git changeset ID we specified (the `git fetch` won't check out the changeset we need of course), so I think it still needs to stay as it is. It's caching fine in my test environment.

Reply:
So you only fetch, not checkout, fair. In that case would you mind extending the comment above to indicate this should also be set back to `false` once the base image is updated?
I only mention it, as this is now different from the Windows target, and that is going to confuse is in the future :D

Reply:
Comment updated.

Reply:
Is the same not needed for the ci-build?
(or, what is an updated vcpkg needed for?)

Reply:
No, the updated vcpkg is only needed for arm64 builds on x86_64 (or building x86_64 on arm64).